src/views/conference/2020/index/index.jsx

```diff
@@ -91,6 +91,15 @@ const ConferenceSplash = () => (
                     <br />
                     <br />
                     <FormattedMessage id="conference-2020.registrationDelayed" />
+                    <br />
```

Comment:
Please add another linebreak here:
```suggestion
                    <br />
                    <br />
```